src/broker/content-cache.c

```diff
@@ -187,14 +177,13 @@ static void cache_entry_destroy (void *arg)
 static struct cache_entry *cache_entry_create (flux_t *h, const char *blobref)
 {
     struct cache_entry *e;
+    int bloblen = strlen (blobref) + 1;
```

Comment:
perhaps add a comment to describe the single malloc for 2 mallocs, since may be non-obvious for a casual reviewer of code (and perhaps add similar comment for `aux_item` change in prior PR too).